nautobot/ui/src/components/common/BSNavBar.jsx

```diff
@@ -1,14 +1,21 @@
-import { Container, Nav, Navbar, NavDropdown } from "react-bootstrap"
-import { faRightToBracket } from "@fortawesome/free-solid-svg-icons";
+import { Button, Container, Nav, Navbar, NavDropdown } from "react-bootstrap"
+import { faSignIn, faSignOut } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { Link, NavLink } from "react-router-dom"
+import {useState, useEffect} from "react"
 import useSWR from "swr"
+import axios from "axios";
 
 
 const fetcher = (url) => fetch(url, { credentials: "include" }).then((res) => res.json());
 
 export default function BSNavBar() {
   const { data, error } = useSWR("/api/get-menu/", fetcher)
+  const [ isLoggedIn, setIsLoggedIn] = useState(false)
+  const logout = () => {
+    axios.get("/api/users/tokens/logout/")
```

Comment:
not using `useSWR` cause this is an operation that would only be done once, and I don't really need the benefits of  `useSWR` here

Reply:
Do we need to use three different methods for fetch? The only reason I used useSWR is because it's recommended by CRA but if Axios is better we could align on that